scripts/superdesk-authoring/authoring.js

```diff
@@ -61,9 +61,9 @@
 
     var DEFAULT_SCHEMA = Object.freeze({
         slugline: {maxlength: 24},
-        headline: {maxlength: 64, order: 1, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor']},
-        abstract: {maxlength: 160, order: 2, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor']},
-        body_html: {order: 5, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor', 'embed', 'picture', 'table']},
```

Comment:
Yes, but that's the way it works, it's fine. All settings are made in json file from where it takes settings, and default_schema is there just in case, as some default settings. If you want to make some changes, they are changed in:
https://github.com/superdesk/superdesk/blob/ntb/server/data/content_types.json